src/nbl/video/IGPUDescriptorSet.cpp

```diff
@@ -45,9 +45,10 @@ asset::IDescriptor::E_TYPE IGPUDescriptorSet::validateWrite(const IGPUDescriptor
     const char* debugName = getDebugName();
 
     // screw it, we'll need to replace the descriptor writing with update templates of descriptor buffer soon anyway
-    const auto descriptorType = getBindingType(write.binding);
+    uint32_t descriptorRedirectBindingIndex;
```

Comment:
its super important to use typed indices, instead of an unsafe `uint32_t` type
https://github.com/Devsh-Graphics-Programming/Nabla/pull/693#discussion_r1659353745